schema/alpha/artifacts/metrics.json

```diff
@@ -0,0 +1,6 @@
+
+{
```

Comment:
Are metrics also a kind of artifact?

Reply:
Strictly speaking metrics is not artifact(no URI), it is easier to handle it as artifact.